tests/transports/tango/test_tango_util.py

```
@@ -0,0 +1,50 @@
+    tango_dev_class_name,
+    tango_dev_name,
+    validate_tango_id,
+)
+
+
+class TestValidateTangoId:
+    @pytest.mark.parametrize(
+        "id",
+        ["DEVICE", "DEV-1", "dev_1", "ALPHA", "BENCHMARK-DEVICE", "0LEAD"],
+    )
+    def test_accepts_valid_ids(self, id: str):
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)